src/components/course/CourseSidebarPrice.jsx

```diff
@@ -33,34 +35,32 @@ const CourseSidebarPrice = () => {
   }
 
   const hasDiscountedPrice = coursePrice.discounted < coursePrice.list;
-  if (hasDiscountedPrice) {
-    return (
-      <>
-        <div className="mb-2">
-          {coursePrice.discounted > 0 ? (
-            <>
-              ${coursePrice.discounted}
-              {!enterpriseConfig?.hideCourseOriginalPrice && (
-                <>
-                  {' '}
-                  <del>${coursePrice.list}</del>
-                </>
-              )}
-              {' '}
-            </>
-          ) : (
-            <>
-              ${coursePrice.discounted} {currency}
-            </>
-          )}
-        </div>
-        <span>Sponsored by {enterpriseConfig.name}</span>
-      </>
-    );
+  if (!hasDiscountedPrice) {
+    return <span>${numberWithPrecision(coursePrice.list)} {currency}</span>;
   }
 
   return (
-    <span>${coursePrice.list} {currency}</span>
+    <>
+      <div className="mb-2">
+        {coursePrice.discounted > 0 ? (
+          <>
+            ${numberWithPrecision(coursePrice.discounted)}
+            {!enterpriseConfig?.hideCourseOriginalPrice && (
+              <>
+                {' '}
+                <del>${numberWithPrecision(coursePrice.list)}</del>
```

Comment:
I think we should use `text-decoration: line-through` on a css class rather than using `<del>` tags? But maybe there's an accessibility consideration I don't know about.

Reply:
ah I do not know much about this part I will check

Reply:
check out the latest commit! tested on screen reader